app/controllers/reviews_controller.rb

```diff
@@ -3,12 +3,15 @@ class ReviewsController < ApplicationController
 
   def create
     @video = Video.find(params[:video_id])
-    review = @video.reviews.build(review_params.merge!(user: current_user))
-    if review.save
-      redirect_to @video
+    review = @video.reviews.create(review_params.merge!(user: current_user))
+    if review.errors.any?
+      review.errors.messages.values.each do |msg|
```

Comment:
Rails can use I18n and locale files to display error messages. I haven't really worked with that but if you'd google around i believe you could find something. Its a better option than hardcoding the message to model. For now if my solution causes you issues just use yours. I haven't really examined that :) Good job anyway.